addons/traefik/disable

```diff
@@ -1,19 +1,16 @@
 #!/usr/bin/env bash
-
 set -e
 
 source $SNAP/actions/common/utils.sh
 CURRENT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
 
 NAMESPACE_PTR="traefik"
 KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
-KUBECTL_DELETE_ARGS="--wait=true --timeout=180s --ignore-not-found=true"
-
+@@ -12,7 +11,7 @@ KUBECTL_DELETE_ARGS="--wait=true --timeout=180s --ignore-not-found=true"
```

Comment:
This looks like an unresolved merge conflict

Reply:
deleted traefik related changed